dotCMS/src/main/java/com/dotcms/cluster/common/ClusterServerActionThread.java

```diff
@@ -56,6 +64,24 @@ public void run() {
 					}
 					connection.commit();
 				}
+
+				if(LicenseUtil.getLevel()== LicenseLevel.COMMUNITY.level){
+				    if(noLicense==0){
+				        noLicense=System.currentTimeMillis();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)